### PR TITLE
feat: ➕ add CamelCase and CamelCaseProperties

### PR DESCRIPTION
**Pull request by Beraliv**
Covers #254

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ If you use any [functions](https://github.com/krzkaczor/ts-essentials/blob/maste
     - DeepNullable
     - DeepUndefinable
     - [Difference between `DeepRequired` and `DeepNonNullable`](#difference-between-deeprequired-and-deepnonnullable)
+  - [String literal types](#string-literal-types)
```
Comment (krzkaczor) [inline README.md:49]: I think this is missing readme entry

Comment (Beraliv) [reply README.md:49]: Ah, right

Let me add the description before merging

File: README.md [hunk 2]
```diff
@@ -46,9 +46,7 @@ If you use any [functions](https://github.com/krzkaczor/ts-essentials/blob/maste
     - DeepNullable
     - DeepUndefinable
     - [Difference between `DeepRequired` and `DeepNonNullable`](#difference-between-deeprequired-and-deepnonnullable)
-  - [String literal types](#string-literal-types)
-    - CamelCase
-    - DeepCamelCaseProperties
+  - [Literal types](https://github.com/krzkaczor/ts-essentials/tree/feat/camel-case/lib/literal-types#literal-types)
```
Comment (Beraliv) [inline README.md:49]: TODO: replace `feat/camel-case` with `master` before merging

File: README.md (hunk 1 above)
Comment (Beraliv) [reply README.md]: Added a separate README for literal types ✅

File: README.md (hunk 2 above)
Comment (krzkaczor) [reply README.md:49]: I'd add here searchable keywords like "camel case".

Comment (Beraliv) [reply README.md:49]: I've added CamelCase ✅

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**